packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -373,9 +373,9 @@ const getFormFieldValue = (
   let tempField: IFormField
   for (const key in sectionDraftData) {
     tempField = sectionDraftData[key] as IFormField
-    return (tempField &&
-      tempField.nestedFields &&
-      tempField.nestedFields[field.name]) as IFormFieldValue
+    if (tempField?.nestedFields?.[field.name]) {
+      return tempField.nestedFields[field.name] as IFormFieldValue
+    }
```

Comment:
This was returning `false` instead of returning the `fieldValue` if found or empty string otherwise.